auditbeat/docs/auditbeat-options.asciidoc

```diff
@@ -0,0 +1,62 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Auditbeat modules. Make sure you keep the
+//// descriptions generic enough to work for all modules. To include
+//// this file, use:
+////
+//// include::{docdir}/auditbeat-options.asciidoc[]
+////
+//////////////////////////////////////////////////////////////////////////
+
+[id="module-standard-options-{modulename}"]
+[float]
+==== Standard configuration options
+
+You can specify the following options for any {beatname_uc} module. 
+
+*`module`*:: The name of the module to run.
+
+*`datasets`*:: A list of datasets to execute. Make sure that you only list datasets that
+are available in the module. It is not possible to reference datasets from
+other modules.
+
+*`enabled`*:: A Boolean value that specifies whether the module is enabled. If you use the
+default config file, +{beatname_lc}.yml+, the System module is enabled (set to
+`enabled: true`) by default. If the `enabled` option is missing from the
+configuration block, the module is enabled by default.
+
+*`period`*:: The frequency at which the datasets check for changes. If a system
+is not reachable, {beatname_uc} returns an error for each period. This setting
+is required.
+ifeval::["{modulename}"=="system"]
+For most datasets, especially `process` and `socket`, a shorter period is recommended.
+endif::[]
```

Comment:
Only the System module has `datasets` and `period` - so maybe we should only include those options for it?